node/src/Pos/Client/CLI/Options.hs

```diff
@@ -188,12 +188,16 @@ webPortOption portNum help =
         <> Opt.value portNum
         <> Opt.showDefault
 
-walletPortOption :: Word16 -> String -> Opt.Parser Word16
-walletPortOption portNum help =
-    Opt.option Opt.auto $
-        templateParser "wallet-port" "PORT" help -- "Port for wallet"
-        <> Opt.value portNum
-        <> Opt.showDefault
+daedalusWalletAddressOption :: Maybe NetworkAddress -> Opt.Parser NetworkAddress
```

Comment:
Why "daedalus" prefix if you make this PR not for daedalus use?

Reply:
@Martoon-00 true, right. I will remove the prefix.